### PR TITLE
Feedback oana 03062020

### PR DESCRIPTION
**Pull request by BastijnKostense**
1. Ask for general comment
2. Ask if the split is relevant for the coronavirus
3. Ask if it was easy to identify a single dominant sentiment in the video
4. Update frontpage to include new questions and remove typos
5. Update training to include new questions
6. Indicate that 4 is neutral on a 7-point Likert scale

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -36,9 +36,14 @@ def assessmentPage():
 def post_javascript_data():
     jsdata = request.form['video_data']
     jsdata = json.loads(jsdata)
+    print(jsdata)
```
Comment (mikebeijen) [inline app.py]: Left over debugging statement?

**Review by mikebeijen — CHANGES_REQUESTED** [review]
On a general note, is there a specific reason why we are using strings index.js to indicate boolean values? Can't we just use bools?

**Comment by BastijnKostense** [comment]
@mikebeijen they are yes/no questions so answering them with true of false might have been ambigious, or at least, less easily recognizable

**Comment by mikebeijen** [comment]
Alright, sounds good! merging then!

**Review by mikebeijen — APPROVED** [review]

**Pull request merged**